splitwise/splitwise.py

```diff
@@ -0,0 +1,42 @@
+from user import User
+from constants import *
+
+class Splitwise:
+    def __init__(self, users_list):
+        self.users_graph = [ [ 0 for _ in range(User.user_count)] for i in range(User.user_count) ]
```

Comment:
are you not storing all the expenses?

Reply:
not a good idea to lose the primary data

Reply:
Oh, should I have used any DB here ?
In that case the data will be lost once we stop our program. My mistake

Reply:
No DB. You could have kept the expenses list in this class.